designs/panorama_plugin/panorama_plugin.md

```diff
@@ -0,0 +1,65 @@
+# Panorama Security Plugin
+This is a security appliance plugin for the Panorama Security Management Appliance, which orchestrates PaloAlto virtual firewalls deployed by OSC. The plugin shall conform to the [security manager api](https://github.com/opensecuritycontroller/security-mgr-api) specification. We propose to have:
+- A Panorama Dynamic address group for each OSC security policy. It will be labeled by a tag and targeted by a _Panorama_ security policy.
+- In Panorama, we define two classes of tags: one to associate Panorama Address Objects with a security group, and another -- with a security policy.
```

Comment:
> In Panorama, we define two classes of tags: one to associate Panorama Address Objects with a security group, and another -- with a security policy. 

with a **OSC** security group and and another set of tags for defining DAG's match criteria.